src/main/scala/io/qbeast/core/transform/CDFQuantilesTransformer.scala

```diff
@@ -64,8 +70,7 @@ trait CDFQuantilesTransformer extends Transformer {
    *
    * @return
    */
-  override def stats: ColumnStats = {
-    ColumnStats(s"${columnName}_quantiles" :: Nil, Nil)
-  }
+  override def stats: ColumnStats =
```

Comment:
Wouldn't be better to just return an object NoColumnStats?

Reply:
Right now, since the `SparkRevisionFactory` is tight to the `ColumnStats` to resolve the `Transformation`, we still need at least the name of the column (even if they are added through the `columnStats` options).

It would be super nice to take a look into the [Revision Flow issue](https://github.com/Qbeast-io/qbeast-spark/issues/223), but not on this PR